.github/workflows/vendoring/unix.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+cd 06-vendoring
+python3.12 -m venv .venv
+source .venv/bin/activate
+.venv/bin/pip install pyodide-build
+.venv/bin/pyodide venv .venv-pyodide
+.venv-pyodide/bin/pip install -t src/vendor -r vendor.txt
+
```

Comment:
We should definitely at the very least have a wrangler deploy dry run, if not a full deploy and a curl